sdk/src/types/block/address/implicit_account_creation.rs

```diff
@@ -11,7 +11,7 @@ use crate::types::block::address::Ed25519Address;
 /// [`AccountOutput`](crate::types::block::output::AccountOutput).
 #[derive(Copy, Clone, Display, Eq, PartialEq, Ord, PartialOrd, Hash, FromStr, AsRef, Deref, From, Packable)]
 #[as_ref(forward)]
-pub struct ImplicitAccountCreationAddress(Ed25519Address);
+pub struct ImplicitAccountCreationAddress(pub(crate) Ed25519Address);
```

Comment:
Maybe just make it pub? Or add a getter?

Reply:
I mean, it doesn't necessarily have to be part of the match

Reply:
Yeah but keep in mind this is supposed to be a public type but it's kind of unusable if you can't access the inner data somehow.

Reply:
It has Deref but fair enough, will add a getter 👍🏻

Reply:
Added a getter